popde/density_estimate.py

```diff
@@ -108,18 +108,83 @@ def prepare_data(self):
             self.transf_data = self.data
 
         if self.stdize:
-            std_transf = ['stdize' for dim in self.ndim]
+            std_transf = ['stdize'] *self.ndim
```

Comment:
```suggestion
            std_transf = ['stdize'] * self.ndim
```
Spaces around operators